NOTES.ZOS

```diff
@@ -0,0 +1,73 @@
+
+ NOTES FOR THE Z/OS PLATFORM
+ ===========================
+
+ First of all note that OpenSSL is currently NOT supported on z/OS.
+
+ However, if you want to build OpenSSL on z/OS anyway, here are a few
+ hints on how to do that. Though these hints are intended to be helpful,
+ they might not work for you at all, depending on your build environment.
+
+ Building OpenSSL 1.1.1
+ ----------------------
+ (with xplink linkage, 64 bit, from EBCDIC sources)
+
+ 1. get a working EBCDIC perl version.
+
+ 2. gunzip the .tar file.
+
+ 3. use pax to extract from .tar. Example:
+
+      pax -ofrom=ISO8859-1,to=IBM-1047 -rvf openssl-1.1.1e.tar
+
+ 4. adjust the configuration files. Examples:
+
+ 4.1 10-main.conf
+
+     "OS390-Unix" => {
+         inherit_from     => [ "BASE_unix", asm("zos_asm") ],
+         cc               => "./tools/c99.sh",
```

Comment:
Value should be `'$(SRCDIR)/util/c99.sh'` to make it possible to build outside of the source tree.